build_runner/lib/src/logging/build_for_input_logger.dart

```diff
@@ -60,15 +60,6 @@ class BuildForInputLogger implements Logger {
     if (logLevel >= Level.SEVERE) {
       _errorWasSeen = true;
     }
-    if (logLevel >= Level.WARNING) {
```

Comment:
I think we still want this - unless we stopped removing the asset that its running on from the description as well (and maybe even then).

Reply:
Header splitting happens where we print now. This code path isn't realistically getting hit now, but if we fixed this we'd end up with a black line before the header, not splitting the header like we intended.

Reply:
ack